data/campaigns/Eastern_Invasion/scenarios/02_The_Escape_Tunnel.cfg

```diff
@@ -1,530 +1,625 @@
-            x,y=22,5
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        [scroll_to]
-            x,y=28,6
-        [/scroll_to]
-        [remove_shroud]
-            side=1
-            x,y=28,6
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        [scroll_to]
-            x,y=34,5
-        [/scroll_to]
-        [remove_shroud]
-            side=1
-            x,y=34,5
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        [message]
-            speaker=Dacyn
-            message= _ "We must make haste to escape before the undead catch up."
-        [/message]
-
-        [message]
-            speaker=Gweddry
-            message= _ "Wait, before we go anywhere — who were those undead?"
-        [/message]
-
-        [message]
-            speaker=Dacyn
-            message= _ "I will explain later. For now, suffice to say they are much too powerful for us; our only hope is to head north."
-        [/message]
-    [/event]
-
-    [event]
-        name=moveto
-        [filter]
-            x,y=29,10
+#
+# Simple and easy, especially compared to S01.
+#
+# This is beatable without any combat or recruiting - just run straight through the dwarfish tunnels, and wade through the river.
+# More ambitious players can take the north path and raid the troll trezzur hole for some gold and a powerful item
+# REALLY ambitous players can fight their way through the troll leader for extra XP.
+#
+
+#define TURNS
+    {ON_DIFFICULTY 15 13 11}#enddef # undead arrival plus 3. I don't like tight turn limits, but couldn't think of another way to discourage farming the undead
+
+    [scenario]
+        id=02_The_Escape_Tunnel
+        name= _ "The Escape Tunnel"
+        map_file=02_The_Escape_Tunnel.map
+        turns={TURNS}
+        next_scenario=03_An_Unexpected_Appearance
+        victory_when_enemies_defeated=no
+
+        {UNDERGROUND}
+
+        [story]
+            [part]
+                story=_ "Screams and shouts echoed across the surface as Gweddry rushed his remaining men into the roughly hewn tunnel. A foul stench wafted up from below..."
+                music=silence.ogg
+                background=story/S02.jpg
+            [/part]
+        [/story]
+        {EI_TRACK {JOURNEY_02_NEW} }
+        {SCENARIO_MUSIC the_deep_path.ogg}
+
+        # wmllint: validate-off
+        [side]
             side=1
-        [/filter]
-
-        [message]
-            speaker=unit
-            image=scenery/signpost.png
-            message= _ "TROL TREZZUR HOLE: KEEP OWT" # wmllint: no spellcheck
-        [/message]
-
-        [allow_undo][/allow_undo]
-    [/event]
-
-    # Find the dwarves
-    [event]
-        name=sighted,moveto
-        [filter]
+            controller=human
+            gold=120
+            team_name,user_team_name=wesnothians,_"Wesnothians"
```

Comment:
common point throughout, but I don't really like this abuse of commas to put multiple keys on the same line

Reply:
Agreed. I think it's convention to have them on separate lines in WML, similar to Python convention.

Reply:
I don't either but it was not possible for me to manually fix all of them. If nobody has a script that can help with separating them, this will have to go very low on the priority list unless someone wants to help with doing it.

Reply:
I'll take a look at automating this.  I guess I'll need to write a script to fix this in my DE WML as well, so I might as well get it over with now and fix EI too.

Reply:
Keep in mind that there are certain sets of keys that are okay on one line, such as `x,y`, `anything_x,anything_y`, or `red,green,blue`. In those cases I'd usually consider it wrong to split them.